src/unlimited.h

```diff
@@ -42,16 +46,17 @@ class CNode;
 class CChainParams;
 
 
-extern uint32_t blockVersion;  // Overrides the mined block version if non-zero
+extern uint32_t blockVersion; // Overrides the mined block version if non-zero
 extern uint64_t maxGeneratedBlock;
 extern unsigned int excessiveBlockSize;
 extern unsigned int excessiveAcceptDepth;
 extern unsigned int maxMessageSizeMultiplier;
 /** BU Default maximum number of Outbound connections to simultaneously allow*/
 extern int nMaxOutConnections;
 
-extern std::vector<std::string> BUComments;  // BU005: Strings specific to the config of this client that should be communicated to other clients
-extern std::string minerComment;  // An arbitrary field that miners can change to annotate their blocks
+extern std::vector<std::string>
+    BUComments; // BU005: Strings specific to the config of this client that should be communicated to other clients
```

Comment:
@gandrewstone : a better Python script might be nice, but meantime this little bash hack job can find all lines in src/*.{h,cpp} with line length exceeding a hard limit AND containing a // comment:

https://gist.github.com/ftrader/f60500881399d155b4c160252ae48be0